.github/workflows/ci.yml

```
@@ -73,9 +73,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: swiftwasm/swiftwasm-action@v5.8
+      - uses: bytecodealliance/actions/wasmtime/setup@v1
+      - uses: swiftwasm/setup-swiftwasm@v1
         with:
-          shell-action: carton test --environment node
+          swift-version: "wasm-5.8.0-RELEASE"
```

Let's bump this while we're here.

```suggestion
          swift-version: "wasm-5.9.2-RELEASE"
```